README.md

```diff
@@ -1,22 +1,22 @@
 # ENVITED OpenMSL Organization
 
-## Public organization profile
+## ASC(S e.V. Governance Rules
 
-The profile information is described in [profile/README.md](profile/README.md) and displayed on the organization main page.
-Read the corresponding GitHub [documentation](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile) for more information on organization profiles.
+The collaboration of contributors in this GitHub organzation is organized under the umbrella of the Automotive Solution Center for Simulation e.V.
+In this community the general [Governance Rules](/doc/governance_rules.md) needed to fulfill industry compliance regulations are set.
```

Comment:
Here are two verbs in this sentence

Reply:
In my opinion this sentence is correct. "are" is the only verb.

Reply:
Now I got it.